openedx/core/djangoapps/profile_images/tests/test_views.py

```diff
@@ -330,34 +335,12 @@ def check_remove_event_emitted(self):
             setting='profile_image_uploaded_at', old=TEST_UPLOAD_DT, new=None
         )
 
-    def test_unsupported_methods(self, mock_log):
-        """
-        Test that GET, PUT, PATCH, and DELETE are not supported.
-        """
-        self.assertEqual(405, self.client.get(self.url).status_code)
-        self.assertEqual(405, self.client.put(self.url).status_code)
-        self.assertEqual(405, self.client.patch(self.url).status_code)
-        self.assertEqual(405, self.client.delete(self.url).status_code)
-        self.assertFalse(mock_log.info.called)
-        self.assert_no_events_were_emitted()
-
-    def test_anonymous_access(self, mock_log):
```

Comment:
I put the squashed the tests together in the Upload test case, but it does test all the methods (POST, DELETE, and even GET, even though there's no actual GET method).